src/test/java/seedu/trackbeau/testutil/PersonUtil.java

```diff
@@ -34,9 +38,15 @@ public static String getPersonDetails(Customer customer) {
         sb.append(PREFIX_PHONE + customer.getPhone().value + " ");
         sb.append(PREFIX_EMAIL + customer.getEmail().value + " ");
         sb.append(PREFIX_ADDRESS + customer.getAddress().value + " ");
-        customer.getTags().stream().forEach(
-            s -> sb.append(PREFIX_TAG + s.tagName + " ")
+        sb.append(PREFIX_SKINTYPE + customer.getSkinType().value + " ");
```

Comment:
Similarly, do we need to rename this class to CustomerUtil as well as its methods e.g. getPersonDetails to getCustomerDetails? Or can this class be a superclass for both Customer and Staff?